media/css/firefox/family-nav.less

```diff
@@ -817,3 +817,29 @@ html[lang^='en'] .fxfamilynav .subsubnav a {
         }
     }
 }
+
+// temporary survey for tertiary nav
+.survey {
+    padding-bottom: 4px;
+    margin: 0 -10px 6px;
+
+    a {
+        display: block;
+        padding: 4px 10px;
+        background: #eee;
+
+        i {
```

Comment:
Might this work just as well with <code>:before</code> instead of an empty element in the HTML?